addons/zookeeper/templates/backupactionset.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
     {{- include "zookeeper.labels" . | nindent 4 }}
 spec:
   backupType: Full
+  env:
+    - name: DP_DB_PORT
```

Comment:
DP_DB_PORT is injected by default

Reply:
Based on my check, this var DP_DB_PORT does not get automatically injected.

Reply:
Redeclare the variable as KB_CLIENT.